### DISCUSSION TITLE
Dag d'ont get scheduled when using timetable for timezone "Europe/Paris" with schedule time before 2 AM

### DISCUSSION THREAD

**Discussion by derrickrose** [apache/airflow#23781] [General]
### Apache Airflow version

2.2.5

### What happened

Dag don't get scheduled when we use a DailyTimetable (a subclass of Timetable) which should trigger our dag every day at 1h15 AM CEST time.

Please note that we force our plugin to use the timezone "Europe/Paris" since other team working with the same shared airflow cluster use the timezone UTC (we cannot set "Europe/Paris" in airflow.cfg file as default_timezone)

### What you think should happen instead

Our dag should get triggered at 1h15 AM CEST time

### How to reproduce

Will complete this in a minute ...

### Operating System

windows

### Versions of Apache Airflow Providers

apache-airflow-providers-amazon==1.3.0
apache-airflow-providers-ftp==2.0.1
apache-airflow-providers-http==1.1.1
apache-airflow-providers-imap==2.0.1
apache-airflow-providers-jenkins==1.1.0
apache-airflow-providers-microsoft-winrm==1.1.0
apache-airflow-providers-odbc==1.0.1
apache-airflow-providers-sqlite==2.0.1

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Please (as advised when you attempted to open the issue) don't open an issue when what you really are about is troubleshooting your configuration/issue rather than reporting and airflow bug. Open discussion instead. 

And you still need to provide more details for anyone to be able to discuss it and help you (possibly - there is no guarantee) - people help here in their free time.
